src/plugins/intel_cpu/src/nodes/executors/acl/acl_eltwise.cpp

```diff
@@ -0,0 +1,173 @@
+            acl_pow->configure(&src1Tensor, &src2Tensor, &dstTensor);
+            break;
+        case Algorithm::EltwiseEqual:
+            if (!NEElementwiseComparison::validate(&src1TensorInfo, &src2TensorInfo, &dstTensorInfo, ComparisonOperation::Equal))
+                return false;
+            acl_comp = std::make_unique<NEElementwiseComparison>();
+            acl_comp->configure(&src1Tensor, &src2Tensor, &dstTensor, ComparisonOperation::Equal);
+        case Algorithm::EltwiseNotEqual:
+            if (!NEElementwiseComparison::validate(&src1TensorInfo, &src2TensorInfo, &dstTensorInfo, ComparisonOperation::NotEqual))
+                return false;
+            acl_comp = std::make_unique<NEElementwiseComparison>();
+            acl_comp->configure(&src1Tensor, &src2Tensor, &dstTensor, ComparisonOperation::NotEqual);
+        case Algorithm::EltwiseGreater:
+            if (!NEElementwiseComparison::validate(&src1TensorInfo, &src2TensorInfo, &dstTensorInfo, ComparisonOperation::Greater))
+                return false;
+            acl_comp = std::make_unique<NEElementwiseComparison>();
+            acl_comp->configure(&src1Tensor, &src2Tensor, &dstTensor, ComparisonOperation::Greater);
+        case Algorithm::EltwiseGreaterEqual:
+            if (!NEElementwiseComparison::validate(&src1TensorInfo, &src2TensorInfo, &dstTensorInfo, ComparisonOperation::GreaterEqual))
+                return false;
+            acl_comp = std::make_unique<NEElementwiseComparison>();
+            acl_comp->configure(&src1Tensor, &src2Tensor, &dstTensor, ComparisonOperation::GreaterEqual);
+        case Algorithm::EltwiseLess:
+            if (!NEElementwiseComparison::validate(&src1TensorInfo, &src2TensorInfo, &dstTensorInfo, ComparisonOperation::Less))
+                return false;
+            acl_comp = std::make_unique<NEElementwiseComparison>();
+            acl_comp->configure(&src1Tensor, &src2Tensor, &dstTensor, ComparisonOperation::Less);
+        case Algorithm::EltwiseLessEqual:
+            if (!NEElementwiseComparison::validate(&src1TensorInfo, &src2TensorInfo, &dstTensorInfo, ComparisonOperation::LessEqual))
+                return false;
+            acl_comp = std::make_unique<NEElementwiseComparison>();
+            acl_comp->configure(&src1Tensor, &src2Tensor, &dstTensor, ComparisonOperation::LessEqual);
+        default:
+            IE_THROW() << "Unsupported operation type for ACL Eltwise executor";
+    }
+    return true;
+}
+
+void AclEltwiseExecutor::exec(const std::vector<MemoryCPtr> &src, const std::vector<MemoryPtr> &dst,
+                              const void *post_ops_data_) {
+    src1Tensor.allocator()->import_memory(src[0]->GetPtr());
+    src2Tensor.allocator()->import_memory(src[1]->GetPtr());
+    dstTensor.allocator()->import_memory(dst[0]->GetPtr());
+    std::cout << static_cast<int>(eltwiseAttrs.algorithm) << std::endl;
+    switch (aclEltwiseAttrs.algorithm) {
```

Comment:
maybe we can add std::function `exec_func` which will wrap different `*->run()` methods on initialization stage? it will allow to have inside exec() method only exec_func() call and avoid additional switch/case block.